scripts/export_inspector_findings.py

```diff
@@ -169,10 +171,20 @@ def column_alpha(self, col: int) -> str:
     def findings_sort(self, item: tuple[str, list[SummaryType]]) -> tuple[int, str]:
         score = 0
         weights = {'HIGH': 1, 'CRITICAL': 10}
-        for summary in item[1]:
+        vulnerability, summaries = item
+        for summary in summaries:
             count = len(summary['resources'])
             score += count * weights.get(summary['severity'], 0)
-        return score, item[0]
+        if vulnerability.startswith('CVE-'):
+            # Best secondary-sorting effort on CVE findings, vulnerability names
+            # not prefixed with 'CVE' may reflect an inaccurate secondary order.
```

Comment:
I think that you meant descending instead of ascending, because we currently sorts like …
![Screenshot 2024-09-26 at 12 30 21 AM](https://github.com/user-attachments/assets/2c35a1d8-3254-4ac6-a2e1-dfbb8b61a4ce)

Is that as expected?